code/cfilearchiver/cfilearchiver.cpp

```diff
@@ -318,24 +318,6 @@ void print_instructions()
 	printf( "Creates a vp archive out of a FreeSpace data tree.\n\n" );
 	printf( "Usage:     cfilearchiver archive_name src_dir\n");
 	printf( "Example:   cfilearchiver freespace /tmp/freespace/data\n\n");
-	printf( "Directory structure options:\n" );
```

Comment:
What exactly makes these unneessary? It seems like it could be useful information, although it seems a bit outdated?